test/Datadog.Trace.ClrProfiler.Managed.Tests/IntegrationSignatureTests.cs

```diff
@@ -10,9 +10,9 @@ public class IntegrationSignatureTests
     {
         private static readonly List<MethodInfo> StaticInstrumentations = new List<MethodInfo>()
         {
-            typeof(AspNetCoreMvc2Integration).GetMethod(nameof(AspNetCoreMvc2Integration.BeforeAction)),
-            typeof(AspNetCoreMvc2Integration).GetMethod(nameof(AspNetCoreMvc2Integration.AfterAction)),
-            typeof(AspNetCoreMvc2Integration).GetMethod(nameof(AspNetCoreMvc2Integration.Rethrow)),
+             typeof(AspNetCoreMvc2Integration).GetMethod(nameof(AspNetCoreMvc2Integration.BeforeAction)),
```

Comment:
Can you fix this small thing? I think there's an extra space now.

Reply:
```suggestion
            typeof(AspNetCoreMvc2Integration).GetMethod(nameof(AspNetCoreMvc2Integration.BeforeAction)),
```